### PR TITLE
Fixes #15536: Updating Apache's reverse proxy configuration to handle HTTPS requests on remote-run API

### PR DESCRIPTION
**Pull request by ArmandSy**
https://issues.rudder.io/issues/15536

### CODE REVIEW

File: relay/sources/apache/rudder-apache-relay-common.conf [hunk 1]
```diff
@@ -75,7 +75,9 @@ Alias /reports /var/rudder/reports/incoming
 </Directory>
 
 # Load relay-api
-WSGIScriptAlias /rudder/relay-api /opt/rudder/share/relay-api/relay-api.wsgi
+#WSGIScriptAlias /rudder/relay-api /opt/rudder/share/relay-api/relay-api.wsgi
```
Comment (amousset) [inline relay/sources/apache/rudder-apache-relay-common.conf]: In fact we need to keep this for old API not migrated to relayd yet.

Comment (amousset) [reply relay/sources/apache/rudder-apache-relay-common.conf]: We may need to make it more precise or put the ProxyPass before to avoid conflict

Comment (peckpeck) [reply relay/sources/apache/rudder-apache-relay-common.conf]: i think the proxypass below is already precise enough

**Comment by ArmandSy** [comment]
Commit modified

**Pull request closed without merging**